test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/StrictMathTest.scala

```diff
@@ -0,0 +1,98 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.lang
+
+import org.junit.Test
+import org.junit.Assert._
+
+import java.lang.{Double => JDouble}
+import java.lang.StrictMath
+
+import org.scalajs.testsuite.utils.AssertExtensions.assertExactEquals
+
+class StrictMathTest {
+
+  @Test def floor(): Unit = {
+    // basic
+    assertExactEquals(5.0, StrictMath.floor(5.0))
+    assertExactEquals(5.0, StrictMath.floor(5.7))
+    assertExactEquals(-6.0, StrictMath.floor(-5.7))
+    assertExactEquals(0.0, StrictMath.floor(0.0))
+    assertExactEquals(-0.0, StrictMath.floor(-0.0))
+    assertExactEquals(0.0, StrictMath.floor(0.5))
+    assertExactEquals(-1.0, StrictMath.floor(-0.5))
+
+    // Special values
```

Comment:
For consistency with other tests, put those first, and also include `+0.0` and `-0.0`.